### ISSUE TITLE
Dispute description not being shown in the dispute list

### ISSUE THREAD

**Issue by facuspagnuolo** [aragon/court-dashboard#328]
See [Usability dispute #5](https://court-usability.aragon.org/disputes/5)

The dispute description appears in the dispute detail view but not in the dispute list view

**Comment by fabriziovigevani** [comment]
Ah in the dispute metadata field we expect also to be the `description` even though is present at the IPFS content. The reason for this is so we don't need to fetch all disputes ipfs content when loading all disputes cards.

So it expects something like this
```
"{ \"description\": \"Sample dispute for Aragon Court testing guide\",  \"metadata\": \"QmbN3uaqRMWypUGKUbjuhL8wCgFXGgfktQgoKhTp6zUm6o/metadata.json\" }"
```

**Comment by sohkai** [comment]
@fabriziovigevani Maybe this is something we can document with https://github.com/aragon/court-dashboard/pull/325?

**Comment by fabriziovigevani** [comment]
Closed by #325
